app/renderer/js/pages/preference/general-section.ts

```diff
@@ -92,6 +92,34 @@ export default class GeneralSection extends BaseSection {
 				</div>
 				<div class="title">${t.__('Advanced')}</div>
 				<div class="settings-card">
+
+					<div class="setting-row" id="app-language">
+						<div class="setting-description">${t.__('App Laguage (requires restart)')}</div>
+						<select id="lang-menu" class="lang-menu">
+							<option value="bg">Bulgarian</option>
```

Comment:
I think it makes sense to store the lang names in some array and call it using a for loop or something. You can make a component for the same.

Reply:
Changes made! Please have a look.

Reply:
This is better.